### DISCUSSION TITLE
Hosts falthrough not working as expected...

### DISCUSSION THREAD

**Discussion by kristvanbesien** [coredns/coredns#6512] [Q&A] [answered]
I created this corefile just to try out how I can combain a hosts plugin  with a forward.
```
example.com {
  metadata
  log
  hosts  {
    192.168.3.10 a.example.com
    192.168.3.15 b.example.com
    192.168.3.17 c.d.example.com
    falltrough
  }
  forward d.example.com 192.168.3.40:5354
  errors
}

d.example.com:54 {
  log
  hosts {
    10.0.4.1 e.d.example.com
  }
}
```

This is to test a scenario where a DNS instance is supposed to first look up in a local file, and if that fails, forward the query to another DNS server.

However, this does not appear to work as expected. When I query for example e.d.example.com from the first instance, I get no answer. The request is not fowarded to the second instance. What am I doing wrong here?

**Comment by chrisohaver** [comment, marked as answer]
fallthrough misspelled?

**Comment by kristvanbesien** [comment]
Yes, that was the case. Missed it as the syntax colouring in vscode somehow made me believe it was recognised as a valid keyword.
